lib/features/auth/presentation/screens/login_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'file:///C:/Users/antho/Desktop/kabod_project/kabod_app/lib/core/presentation/res/constants.dart';
+import 'file:///C:/Users/antho/Desktop/kabod_project/kabod_app/lib/features/auth/presentation/components/login_form.dart';
+
+class LoginScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
```

Comment:
Over here, you have too much code altogether.
Try to encapsulate pieces of code into functions so any other developer could understand it better and faster like this:
```suggestion
        body: Stack(
        children: [
          ...getBackground(),
          getLogo(context),
          getLoginForm(),
        ],
      ),
```
It's the magic of Flutter. You can restructure your Widgets Tree easily and in a clean way.

Remember, in each function, you will return the respective Widget:
```
Container getLogo(BuildContext context) {
    return Container(
          alignment: Alignment.topCenter,
          child: Image.asset(
            'images/logo_white.png',
            width: MediaQuery.of(context).size.width * 0.3,
          ),
        );
  }
```